doc/specs/vulkan/chapters/fragops.txt

```diff
@@ -125,26 +125,22 @@ pname:discardRectangleCount)#.
   * [[VUID-vkCmdSetDiscardRectangleEXT-None-00583]]
     The currently bound graphics pipeline must: have been created with the
     ename:VK_DYNAMIC_STATE_DISCARD_RECTANGLE_EXT dynamic state enabled
-  * [[VUID-vkCmdSetDiscardRectangleEXT-firstDiscardRectangle-00584]]
-    pname:firstDiscardRectangle must: be less than
-    sname:VkPhysicalDeviceDiscardRectanglePropertiesEXT.pname:maxDiscardRectangles
   * [[VUID-vkCmdSetDiscardRectangleEXT-firstDiscardRectangle-00585]]
     The sum of pname:firstDiscardRectangle and pname:discardRectangleCount
-    must: be between `1` and
-    sname:VkPhysicalDeviceDiscardRectanglePropertiesEXT::pname:maxDiscardRectangles,
-    inclusive
-  * [[VUID-vkCmdSetDiscardRectangleEXT-pDiscardRectangles-00586]]
-    pname:pDiscardRectangles must: be a pointer to an array of
-    pname:discardRectangleCount valid sname:VkRect2D structures
+    must: be less than or equal to
+    slink:VkPhysicalDeviceDiscardRectanglePropertiesEXT::pname:maxDiscardRectangles
```

Comment:
FYI: Similar change was found objectionable by @jeffbolznv in [#588](https://github.com/KhronosGroup/Vulkan-Docs/pull/588#discussion-diff-144866676).